### PR TITLE
Add option to not run the SOCKS server: `ENABLE_SOCKS_SERVER`

### PR DESCRIPTION
**Pull request by tastelessjolt**
- I would like to not run the SOCKS server since I don't use it.
- So, this PR adds an option `ENABLE_SOCKS_SERVER` to control whether the SOCKS server is run or not.
- Please let me know there is any issue with the variable naming or the implementation.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ ENV CONNECTION_TYPE=tcp
 ENV LAN_NETWORK=
 ENV CREATE_TUN_DEVICE=
 ENV ENABLE_MASQUERADE=
+ENV ENABLE_SOCKS_SERVER=true
```
Comment (ilteoood) [inline Dockerfile]: Let's disable it by default

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ The container is configurable using different environment variables:
 |LAN_NETWORK|No|Lan network used to access the web ui of attached containers. Can be comma seperated for multiple subnets Comment out or leave blank: example 192.168.0.0/24|
 |CREATE_TUN_DEVICE|No|Creates the TUN device, useful for NAS users|
 |ENABLE_MASQUERADE|No|Masquerade NAT allows you to translate multiple IP addresses to another single IP address.|
+|ss|No|Control whether the SOCKS server for the VPN is run or not(default: run)|
```
Comment (ilteoood) [inline README.md]: The env name here is wrong, isn't it?

**Review by ilteoood — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (tastelessjolt) [reply Dockerfile]: Sure, disabled it by default

File: README.md (hunk 2 above)
Comment (tastelessjolt) [reply README.md]: My bad, typo went through, fixed it~

**Review by ilteoood — APPROVED** [review]
Thank you so much for the contribution, I really appreciate it

**Pull request merged**